infrastructure/tooling/src/smoketest/checks/dependencies.js

```diff
@@ -0,0 +1,66 @@
+const taskcluster = require('taskcluster-client');
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Check dependencies',
+  requires: [],
+  provides: [
+    'dependency smoketest',
```

Comment:
```suggestion
    'target-dependencies',
```

This means we can run `yarn smoketest --target dependencies` to run just this check